lib/membrane/rtp/payload_format.ex

```diff
@@ -0,0 +1,128 @@
+defmodule Membrane.RTP.PayloadFormat do
+  @moduledoc """
+  This module contains utilities for resolving RTP default payload types,
+  encoding names, clock rates and (de)payloaders.
+  """
+
+  alias Membrane.RTP
+
+  require Membrane.RTP
+
+  @app :membrane_rtp_format
+
+  @payload_types_specs %{
+    0 => %{encoding_name: :PCMU, clock_rate: 8000},
+    3 => %{encoding_name: :GSM, clock_rate: 8000},
+    4 => %{encoding_name: :G732, clock_rate: 8000},
+    5 => %{encoding_name: :DVI4, clock_rate: 8000},
+    6 => %{encoding_name: :DVI4, clock_rate: 16000},
+    7 => %{encoding_name: :LPC, clock_rate: 8000},
+    8 => %{encoding_name: :PCMA, clock_rate: 8000},
+    9 => %{encoding_name: :G722, clock_rate: 8000},
+    10 => %{encoding_name: :L16, clock_rate: 44100},
+    11 => %{encoding_name: :L16, clock_rate: 44100},
+    12 => %{encoding_name: :QCELP, clock_rate: 8000},
+    13 => %{encoding_name: :CN, clock_rate: 8000},
+    14 => %{encoding_name: :MPA, clock_rate: 90000},
+    15 => %{encoding_name: :G728, clock_rate: 8000},
+    16 => %{encoding_name: :DVI4, clock_rate: 11025},
+    17 => %{encoding_name: :DVI4, clock_rate: 22050},
+    18 => %{encoding_name: :G729, clock_rate: 8000},
+    25 => %{encoding_name: :CELB, clock_rate: 90000},
+    26 => %{encoding_name: :JPEG, clock_rate: 90000},
+    28 => %{encoding_name: :NV, clock_rate: 90000},
+    31 => %{encoding_name: :H261, clock_rate: 90000},
+    32 => %{encoding_name: :MPV, clock_rate: 90000},
+    33 => %{encoding_name: :MP2T, clock_rate: 90000},
+    34 => %{encoding_name: :H263, clock_rate: 90000}
+  }
+
+  @enforce_keys [:encoding_name]
+  defstruct @enforce_keys ++ [payload_type: nil, payloader: nil, depayloader: nil]
+
+  @type t :: %__MODULE__{
+          encoding_name: RTP.encoding_name_t(),
```

Comment:
Do we need this field? If I understand it properly we do two types of registration:

`{:format, encoding_name} => %__MODULE__{}` - in this case we don't need encoding name in struct
and 
`{:payload_type, payload_type} => {encoding_name, clock_rate}` - here we don't use struct

Reply:
I don't insist on this form, but here's the intention: we have a _set_ of registered formats, that should be searchable via encoding_name, and _mapping_ from payload type to some parameters. The mapping is intended to be used just to get assigned value, while the set of formats may be used to obtain the format and pass it somewhere